hack/test/unit-run.go

```diff
@@ -0,0 +1,220 @@
+///usr/bin/env true; exec /usr/bin/env go run "$0" "$@"
```

Comment:
This is a little shebang hack, which lets `go` files be executable as `./hack/test/unit-run.go`.
So developers can run (json-formatted) unit tests directly bypassing `gotestsum` by `./hack/test/unit-run.go`. It would be more useful if this script would support more functionality, not sure if it's best to remove it or leave it.